Source/CombatExtended/CombatExtended/Comps/CompProperties_Fragments.cs

```diff
@@ -12,6 +12,8 @@ public class CompProperties_Fragments : CompProperties
     {
         public List<ThingDefCountClass> fragments = new List<ThingDefCountClass>();
         public float fragSpeedFactor = 1f;
+        public float fragShadowChance = 0.2f;
+        public FloatRange fragAngleRange = new FloatRange(0.5f, 5f);
```

Comment:
Why 0.5° to 5° instead of the previous 10° to 20°?  This looks like it will be much flatter than previously.  
While we're making changes, do we want to add support for shaped charges and replace the static `Rand.Range(0, 360)` with a second parameterized range?

Reply:
>Why 0.5° to 5° instead of the previous 10° to 20°? This looks like it will be much flatter than previously.

That's the point: I've had several times where large fragments (often from doomsdays) just fly over the heads of someone at the highest point of the arc, which was gimping the damage output.

> While we're making changes, do we want to add support for shaped charges and replace the static `Rand.Range(0, 360) `with a second parameterized range?

Sure.

Reply:
Sounds fine.  Note that you'll need to apply the 0-360 variable range off the projectile's forward direction, rather than a fixed angle.